sdk/storage/azure-storage-blobs/inc/azure/storage/blobs/blob_responses.hpp

```diff
@@ -68,40 +68,40 @@ namespace Azure { namespace Storage { namespace Blobs {
       std::string LeaseId;
     };
 
-    class StartCopyBlobResult : public Azure::Core::Operation<GetBlobPropertiesResult> {
-    public:
-      std::string RequestId;
-      Azure::Core::ETag ETag;
-      Azure::Core::DateTime LastModified;
-      std::string CopyId;
-      Models::CopyStatus CopyStatus;
-      Azure::Core::Nullable<std::string> VersionId;
+  } // namespace Models
 
-    public:
-      GetBlobPropertiesResult Value() const override { return m_pollResult; }
+  class StartCopyBlobOperation : public Azure::Core::Operation<Models::GetBlobPropertiesResult> {
+  public:
+    std::string RequestId;
+    Azure::Core::ETag ETag;
+    Azure::Core::DateTime LastModified;
+    std::string CopyId;
+    Models::CopyStatus CopyStatus;
+    Azure::Core::Nullable<std::string> VersionId;
 
-      ~StartCopyBlobResult() override {}
+  public:
+    Models::GetBlobPropertiesResult Value() const override { return m_pollResult; }
 
-    private:
-      std::string GetResumeToken() const override
-      {
-        // Not supported
-        std::abort();
-      }
+    ~StartCopyBlobOperation() override {}
 
-      std::unique_ptr<Azure::Core::Http::RawResponse> PollInternal(
-          Azure::Core::Context& context) override;
+  private:
+    std::string GetResumeToken() const override
+    {
+      // Not supported
+      std::abort();
```

Comment:
I think this should return the URL address that we use for polling.

Reply:
The URL itself isn't of much use. The resume token will need to include authentication information, and that's hard.